### ISSUE TITLE
default shortcuts and autocomplete disabled after ActualVim

### ISSUE THREAD

**Issue by jackaccyou** [lunixbochs/ActualVim#117]
After i installed actualvim, the shortcuts are not working anymore, e.g. Ctrl+b for build. The autocomplete and the auto closing brackets functions are also missing. Is that something i did wrong or is it the way ActualVim works? thx

**Comment by lunixbochs** [comment]
overlaps these issues:
https://github.com/lunixbochs/ActualVim/issues/105 (key mapping)
https://github.com/lunixbochs/ActualVim/issues/94 (snippets)
https://github.com/lunixbochs/ActualVim/issues/97 (possible fix)
https://github.com/lunixbochs/ActualVim/issues/57 (auto completion)

**Comment by jackaccyou** [comment]
thx so much for the detailed reply, I really appreciate it. But can you elaborate more about the key mapping? I have read the issues already. 
{ "keys": ["ctrl+s"], "context": [{"key": "setting.actual_mode", "operand": true}],  "command": "actual_disable"},
{ "keys": ["ctrl+s"], "context": [{"key": "setting.actual_mode", "operand": false}], "command": "actual_enable"},

so if i press ctrl+s in sublime, it will diable vim and enable all sublime original keymaps?

**Comment by lunixbochs** [comment]
That's the idea, yes. You can also use "Disable ActualVim" from the command palette.

**Comment by lunixbochs** [comment]
You can bypass some keyboard shortcuts per mode in the 0.9.12 release coming out soon.

Basically set settings.vim.modes.normal.av:ctrl+b to `true` in one of the modes in ActualVim settings:
```
    "settings": {
        "vim": {
            "modes": {
                "normal": {
                    "av:ctrl+b": true,
                },
            }
        }
    }```

**Comment by lunixbochs** [comment]
With key mapping fixed, I'm closing this as a duplicate of #97
